Formula/mxnet.rb

```diff
@@ -0,0 +1,35 @@
+class Mxnet < Formula
+  desc "Flexible and efficient library for deep learning"
+  homepage "https://mxnet.apache.org"
+  url "https://dlcdn.apache.org/incubator/mxnet/1.9.1/apache-mxnet-src-1.9.1-incubating.tar.gz"
+  sha256 "11ea61328174d8c29b96f341977e03deb0bf4b0c37ace658f93e38d9eb8c9322"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "python@3.10" => :build
+  depends_on arch: :x86_64
```

Comment:
Is there a tracking issue for ARM support?

Reply:
https://github.com/apache/incubator-mxnet/issues/20028

Reply:
Going to try `-DUSE_SSE=OFF` for ARM